src/index.ts

```diff
@@ -0,0 +1,99 @@
+import {createServer, IncomingMessage, ServerResponse} from 'http'
+import * as fs from 'fs'
+import 'body-parser'
+import {keys} from "./keys"
```

Comment:
Use single quotes instead of double.
Add spaces around the "keys" word in the brackets.